### PR TITLE
Integrations update

### LINKED ISSUE #283: Update Integrations section
cf https://github.com/dobtco/screendoor-v2/pull/3385/commits/7c4f7c435542a4d6e62a12cabe030dc2391c4ead

### PR DESCRIPTION
**Pull request by kariannemah**
Closes #283.

### CODE REVIEW

File: contents/articles/screendoor/integrations/ckan.md [hunk 1]
```diff
@@ -6,7 +6,31 @@ app_pages:
 ---
 
 <div class='alert'>
-  **Note**: CKAN data sync is currently in private beta. [Contact us](mailto:support@dobt.co) for more information.
+    **Note**: CKAN data sync is only available for Enterprise plans.
 </div>
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/opCt0N8jORg" frameborder="0" allowfullscreen></iframe>
+Instead of manually releasing open data to your CKAN portal, Screendoor can sync responses to your forms with a CKAN data set automatically. Revisions to responses can even be reflected in your portal in real time.
```
Comment (ajb) [inline contents/articles/screendoor/integrations/ckan.md]: "responses to your forms" is a bit awkward. Maybe "sync your responses"?

Also, I think it should be stylized as "dataset" as per http://docs.ckan.org/en/ckan-1.8/domain-model-dataset.html

Comment (ajb) [inline contents/articles/screendoor/integrations/ckan.md]: Oop, didn't realize this was existing language.

File: contents/articles/screendoor/integrations/socrata.md [hunk 2]
```diff
@@ -13,22 +13,28 @@ Instead of manually releasing open data to your Socrata portal, Screendoor can s
...
 
-Fill in the web address of your Socrata data portal and press the "Connect to Socrata" button. If you're already signed in, you will be asked whether you want to connect to Screendoor. Press the "Allow" button to complete the process.
+Fill in the URL of your Socrata data portal and press the "Connect to Socrata" button.
 
-![Socrata's authorization page.](../images/socrata_2.png)
+![The Connect to Socrata page.](../images/socrata_2.png)
+
+ If you're already signed in, you will be asked whether you want to connect to Screendoor. Press "Allow" to complete the process.
+
+![Socrata's authorization page.](../images/socrata_3.png)
```
Comment (ajb) [inline contents/articles/screendoor/integrations/socrata.md]: I feel like we can cut this screenshot.

**Comment by ajb** [comment]
Just those couple of inline comments, looks :+1: otherwise. I think we need to make sure that we don't include _too_ many screenshots in these articles, since it could make them harder to follow along with

**Comment by kariannemah** [comment]
kk, addressed your comments. And noted on keeping the number of screenshots down.

**Comment by ajb** [comment]
Great, thanks! Looks great. 

Also, cc @jqnatividad: Joel, here are some updated docs for our CKAN integration, which will be helpful for the extension catalog.

**Pull request merged**